Realm.xcodeproj/project.pbxproj

```diff
@@ -246,6 +236,18 @@
 		3FEB38401E70AC8800F22712 /* ObjectCreationTests.mm in Sources */ = {isa = PBXBuildFile; fileRef = 3FEB383C1E70AC6900F22712 /* ObjectCreationTests.mm */; };
 		3FEC4A3F1BBB18D400F009C3 /* SwiftSchemaTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 3FEC4A3D1BBB188B00F009C3 /* SwiftSchemaTests.swift */; };
 		3FF3FFAF1F0D6D6400B84599 /* KVOTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 5D660FFF1BE98D880021E04F /* KVOTests.swift */; };
+		49B299882409338200B7319C /* RLMAppCredentials.mm in Sources */ = {isa = PBXBuildFile; fileRef = 49B299842409338200B7319C /* RLMAppCredentials.mm */; };
+		49B299892409338200B7319C /* RLMAppCredentials.h in Headers */ = {isa = PBXBuildFile; fileRef = 49B299852409338200B7319C /* RLMAppCredentials.h */; settings = {ATTRIBUTES = (Public, ); }; };
+		49B2998A2409338200B7319C /* RLMApp.mm in Sources */ = {isa = PBXBuildFile; fileRef = 49B299862409338200B7319C /* RLMApp.mm */; };
```

Comment:
The newly added source files need to be added to the Realm iOS Static target in addition to the Realm target.